content/webapp/components/AudioPlayerNew/AudioPlayer.tsx

```diff
@@ -0,0 +1,367 @@
+import * as prismic from '@prismicio/client';
+import {
+  FunctionComponent,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import styled from 'styled-components';
+
+import { font } from '@weco/common/utils/classnames';
+import { dasherize } from '@weco/common/utils/grammar';
+import { AppContext } from '@weco/common/views/components/AppContext/AppContext';
+import CollapsibleContent from '@weco/common/views/components/CollapsibleContent';
+import PrismicHtmlBlock from '@weco/common/views/components/PrismicHtmlBlock/PrismicHtmlBlock';
+import Space from '@weco/common/views/components/styled/Space';
+import { useAVTracking } from '@weco/content/hooks/useAVTracking';
+
+import { formatPlayerTime } from './AudioPlayer.formatters';
+import {
+  PauseIcon,
+  PlayIcon,
+  SkipBackIcon,
+  SkipForwardIcon,
+} from './AudioPlayer.Icons';
+import PlayRate from './AudioPlayer.PlayRate';
+import Scrubber from './AudioPlayer.Scrubber';
+
+const AudioPlayerWrapper = styled.figure<{ $isDark: boolean }>`
+  padding: 1rem;
+  background: ${props =>
+    props.$isDark ? props.theme.color('black') : props.theme.color('white')};
+  margin: 0;
+`;
+
+type PlayPauseButtonProps = { $isPlaying: boolean };
+const PlayPauseButton = styled.button.attrs<PlayPauseButtonProps>(props => ({
+  'aria-pressed': props.$isPlaying,
+}))<PlayPauseButtonProps>``;
+
+const TimeWrapper = styled.div.attrs({
+  className: font('intr', 5),
+})<{ $isDark: boolean }>`
+  display: flex;
+  justify-content: space-between;
+  font-variant-numeric: tabular-nums;
+  color: ${props =>
+    props.$isDark ? props.theme.color('white') : props.theme.color('black')};
+`;
+
+const SkipPlayWrapper = styled.div`
+  grid-column: 2;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const SkipButton = styled.button<{ $isDark: boolean }>`
+  color: ${props =>
+    props.$isDark ? props.theme.color('yellow') : props.theme.color('black')};
+
+  transition:
+    color 0.2s ease-out,
+    transform 0.2s ease-out;
+
+  &:hover {
+    color: ${props =>
+      props.$isDark ? props.theme.color('white') : props.theme.color('black')};
+    transform: scale(1.1);
+  }
+`;
+
+const PlayerRateWrapper = styled.div`
+  grid-column: 3;
+  display: flex;
+  align-items: center;
+  justify-content: end;
+`;
+
+const PlayPauseInner = styled.div<{ $isDark: boolean }>`
+  color: ${props => props.theme.color('yellow')};
+  transition:
+    color 0.2s ease-out,
+    transform 0.2s ease-out;
+
+  .icon__playpause {
+    transition: fill 0.2s ease-out;
+  }
+
+  &:hover {
+    color: ${props =>
+      props.$isDark ? props.theme.color('white') : props.theme.color('black')};
+    transform: scale(1.1);
+
+    .icon__playpause {
```

Comment:
Couldn't think of a different/better way to change the colour of two different parts of the svg than give them specific classes to target (another reason to treat them differently from the other icons)